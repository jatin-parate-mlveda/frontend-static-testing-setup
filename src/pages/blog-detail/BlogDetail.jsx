import BlogTitle from './BlogTitle';

export const BlogDetail = () => (
  <div>
    <BlogTitle title='My Blog' />
    <hr />
    <div>Content area</div>
  </div>
);
