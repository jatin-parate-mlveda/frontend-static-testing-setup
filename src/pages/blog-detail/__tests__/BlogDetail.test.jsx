import { render } from '@testing-library/react';
import { BlogDetail } from '../BlogDetail';

test('matches snapshot', () => {
  const { container } = render(<BlogDetail />);

  expect(container.outerHTML).toMatchSnapshot();
});
