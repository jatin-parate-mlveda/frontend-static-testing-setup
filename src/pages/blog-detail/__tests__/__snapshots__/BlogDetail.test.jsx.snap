// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `"<div><div><div class=\\"container\\">My Blog</div><hr><div>Content area</div></div></div>"`;
